dynatrace/stackstate_checks/dynatrace/dynatrace.py

```diff
@@ -158,16 +237,26 @@ def _collect_topology(self, response, component_type, instance_info):
         :param instance_info: instance configuration
         :return: create the component on stackstate API
         """
+        # since custom device response returns entities
```

Comment:
Move this to `process_custom_device_topology` then you don't have to pollute `_collect_topology` too much

Reply:
done.